src/pages/index/bookmarks.vue

```diff
@@ -58,7 +58,8 @@ export default {
   computed: {
     ...mapGetters(['getUserBookmarks']),
     items() {
-      return this.getUserBookmarks.map(referrer => ({ referrer }));
+      const items = this.getUserBookmarks.map(referrer => ({ referrer }));
+      return items.reverse();
```

Comment:
Not reverse in API?

Reply:
seems presentation requirement, so do in UI for now